code/modules/mob/living/carbon/xenomorph/mark_menu.dm

```diff
@@ -109,6 +109,8 @@
 	switch(action)
 		if("choose_mark")
 			var/selected_type = text2path(params["type"])
+			if(!istype(selected_type, /datum/xeno_mark_define/)) // Hacky fix
+				message_staff("[key_name(usr)] has somehow entered an invalid path while choosing a mark. Possibly an attempted exploit.")
```

Comment:
please get rid of the staff message and just early return here

Reply:
also remove the / at the end of your typepath, make it /datum/xeno_mark_define instead

Reply:
No staff message? I say possibly in the message, but the odds are that it's deliberate. The menu only shows subtypes of /datum/xeno_mark_define

Reply:
It's a useless message. Anyone who is able to check the code will see that it notifies staff members, so the only time this'd flag up is if someone found a bug. At that point, it should just runtime or send a log, not a staff message